platforms/hyperledger-fabric/configuration/roles/create/channel_artifacts/tasks/main.yaml

```diff
@@ -76,6 +76,34 @@
   tags:
     - molecule-idempotence-notest
 
+- name: "Check if genesis block has already been written to Vault"
+  shell: |
+    vault kv get -field=genesisBlock_{{ channel_name }} secret/crypto/ordererOrganizations
```

Comment:
are genesis blocks dependent on channel name?

Reply:
As per [this documentation](https://hyperledger-fabric.readthedocs.io/en/release-2.2/channels.html), there is a genesis block for each channel (as it is a 'subchain').  So far we only create networks with one channel, but for future we want to seperate genesis blocks for each channel, correct?

Reply:
This is the channel genesis. (in the documentation you mentioned)
You have put the correct genesis (network genesis) in the vault, but the channel suffix should be removed as the network genesis is just one file

Reply:
Misunderstanding on my part. Have removed the channel name in commit [ad25b2c](https://github.com/hyperledger-labs/blockchain-automation-framework/pull/1053/commits/ad25b2c131b4539efa614b1f5f263c972b7bf629)